tar.opam

```diff
@@ -8,11 +8,8 @@ dev-repo:     "https://github.com/mirage/ocaml-tar.git"
 doc:          "https://mirage.github.io/ocaml-tar/"
 
 build: [
-  [ "jbuilder" "build" "--only-packages=tar" ]
-]
-
-build-test: [
```

Comment:
this now means that the tests are never executed on travis!?

Reply:
ups, sorry -- tests seem to be executed in the `tar-mirage` opam -- do the tests depend on tar-mirage?

Reply:
The `lib_test/parse_test.ml` uses `Tar_mirage` (not ideal, probably the tests should be split up)